src/server/handlers/github-auth.js

```diff
@@ -66,7 +66,7 @@ export const verify = (req, res, next) => {
     logger.info('User successfully authenticated');
 
     // Redirect to logged in URL
-    res.redirect('/dashboard');
+    res.redirect('/dashboard/select-repositories');
```

Comment:
We don't technically need this change. Eventually, the user will only be redirected to the select-repositories view if they have no enabled repositories.

The reason I made this change now is that it will allow us to validate the "initial login" path before the smarter redirect is in-place. Our designers can see a more complete picture of the site before the "enabled-repositories" dashboard view is finished.

It's a small, meaningless change but I think it speaks a lot.